python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_assets.py

```diff
@@ -1008,6 +1016,18 @@ def test_get_asset_key_not_found(self, graphql_context: WorkspaceRequestContext,
         assert result.data
         snapshot.assert_match(result.data)
 
+    def test_additional_required_keys_query(self, graphql_context: WorkspaceRequestContext):
+        result = execute_dagster_graphql(
+            graphql_context,
+            ADDITIONAL_REQUIRED_KEYS_QUERY,
+            variables={
+                "assetKeys": [{"path": ["int_asset"]}],
```

Comment:
you can add an invalid key here and see if it failed before your change to see if you need the `has` check